vector/src/androidTest/java/im/vector/app/ui/robot/settings/SettingsNotificationsRobot.kt

```diff
@@ -21,22 +21,27 @@ import com.adevinta.android.barista.interaction.BaristaClickInteractions.clickOn
 import im.vector.app.BuildConfig
 import im.vector.app.R
 import im.vector.app.espresso.tools.clickOnPreference
+import im.vector.app.features.VectorFeatures
 
 class SettingsNotificationsRobot {
 
     fun crawl() {
-        if (BuildConfig.USE_NOTIFICATION_SETTINGS_V2) {
-            clickOn(R.string.settings_notification_default)
-            pressBack()
-            clickOn(R.string.settings_notification_mentions_and_keywords)
-            // TODO Test adding a keyword?
-            pressBack()
-            clickOn(R.string.settings_notification_other)
-            pressBack()
-        } else {
-            clickOn(R.string.settings_notification_advanced)
-            pressBack()
+        when (BuildConfig.NOTIFICATION_SETTINGS_VERSION!!) {
```

Comment:
Why not using a `DefaultVectorFeatures` instance here? It would prevent the usage of `!!` (I think)

Reply:
good point! I wasn't sure how the injections would work in the `SanityTests`, I would invest more time but this is removed in #4627